soh/soh/UIWidgets.cpp

```diff
@@ -170,7 +170,6 @@ namespace UIWidgets {
             ImGui::RenderCheckMark(window->DrawList, check_bb.Min + ImVec2(pad, pad), check_col, square_sz - pad * 2.0f);
         } else if ((!disabled && !*v && renderCrossWhenOff) || (disabled && disabledGraphic == CheckboxGraphics::Cross)) {
             const float pad = ImMax(1.0f, IM_FLOOR(square_sz / 6.0f));
-            RenderCross(window->DrawList, check_bb.Min + ImVec2(pad, pad), disabled ? cross_col : check_col, square_sz - pad * 2.0f);
```

Comment:
I'd say that it's better to just comment it out in case we want to do something in the future with it.

Reply:
I'm OK with that.